### ISSUE TITLE
Invalid "DISPLAY" variable

### ISSUE THREAD

**Issue by simonzhaoms** [mlhubber/mlhub#17]
If no graphical display, it would be better to prompt similar messages as 'ml display rain-tomorrow' which is 'Graphic display not available'.

```console
simon@mlhub-test:~$ ml demo clothes-recommender
...     
[InProgress] Writing images to /home/simon/.mlhub/clothes-recommender/results/fashionTexture/visualizations_weightedl2/demo_results/
An error was encountered:

Selected CPU as the process wide default device.
/anaconda/envs/py35/lib/python3.5/site-packages/sklearn/base.py:311: UserWarning: Trying to unpickle estimator LinearSVC from version 0.18.1 when using version 0.19.1. This might lead to breaking code or invalid results. Use at your own risk.
  UserWarning)
/anaconda/envs/py35/lib/python3.5/site-packages/sklearn/base.py:311: UserWarning: Trying to unpickle estimator LabelEncoder from version 0.18.1 when using version 0.19.1. This might lead to breaking code or invalid results. Use at your own risk.
  UserWarning)
/anaconda/envs/py35/lib/python3.5/site-packages/sklearn/base.py:311: UserWarning: Trying to unpickle estimator _SigmoidCalibration from version 0.18.1 when using version 0.19.1. This might lead to breaking code or invalid results. Use at your own risk.
  UserWarning)
/anaconda/envs/py35/lib/python3.5/site-packages/sklearn/base.py:311: UserWarning: Trying to unpickle estimator CalibratedClassifierCV from version 0.18.1 when using version 0.19.1. This might lead to breaking code or invalid results. Use at your own risk.
  UserWarning)
Traceback (most recent call last):
  File "demo.py", line 175, in <module>
    pltAxes = [plt.subplot(1, 3, i+1) for i in range(3)]
  File "demo.py", line 175, in <listcomp>
    pltAxes = [plt.subplot(1, 3, i+1) for i in range(3)]
  File "/anaconda/envs/py35/lib/python3.5/site-packages/matplotlib/pyplot.py", line 1071, in subplot
    fig = gcf()
  File "/anaconda/envs/py35/lib/python3.5/site-packages/matplotlib/pyplot.py", line 601, in gcf
    return figure()
  File "/anaconda/envs/py35/lib/python3.5/site-packages/matplotlib/pyplot.py", line 548, in figure
    **kwargs)
  File "/anaconda/envs/py35/lib/python3.5/site-packages/matplotlib/backend_bases.py", line 161, in new_figure_manager
    return cls.new_figure_manager_given_figure(num, fig)
  File "/anaconda/envs/py35/lib/python3.5/site-packages/matplotlib/backend_bases.py", line 167, in new_figure_manager_given_figure
    canvas = cls.FigureCanvas(figure)
  File "/anaconda/envs/py35/lib/python3.5/site-packages/matplotlib/backends/backend_qt5agg.py", line 24, in __init__
    super(FigureCanvasQTAgg, self).__init__(figure=figure)
  File "/anaconda/envs/py35/lib/python3.5/site-packages/matplotlib/backends/backend_qt5.py", line 234, in __init__
    _create_qApp()
  File "/anaconda/envs/py35/lib/python3.5/site-packages/matplotlib/backends/backend_qt5.py", line 125, in _create_qApp
    raise RuntimeError('Invalid DISPLAY variable')
RuntimeError: Invalid DISPLAY variable
```

**Comment by simonzhaoms** [comment]
```console
simon@mlhubtest:~$ ml display rain-tomorrow
Graphic display not available.

You may like to predict if it will rain tomorrow:
 
  $ ml score rain-tomorrow
```

**Comment by gjwgit** [comment]
DESCRIPTION.yaml contains meta -> display to list the commands that require DISPLAY to be available. Check if DISPLAY not available, print informative message for mlhub user.

**Comment by gjwgit** [comment]
Resolved 16bcc008f545f8d5647d754d7f5612a056412aac
